write-fonts/src/tables/glyf.rs

```diff
@@ -172,24 +184,18 @@ impl SimpleGlyph {
                     .ok_or(BadKurbo::MissingMove)?
                     .quad_to(*p0, *p1),
                 kurbo::PathEl::CurveTo(_, _, _) => return Err(BadKurbo::HasCubic),
-                // I think we can just ignore this, and remove duplicate points
-                // at the end?
-                kurbo::PathEl::ClosePath => (),
+                kurbo::PathEl::ClosePath => {
+                    // remove last point in closed path if has same coords as the move point
+                    let contour = current.as_mut().ok_or(BadKurbo::MissingMove)?;
+                    if contour.len() > 1 && contour.last().unwrap() == contour.first().unwrap() {
```

Comment:
this should work without needing the unwraps, since `Option<T>` impls `PartialEq` if `T` does, and we know they must exist:
```suggestion
                    if contour.len() > 1 && contour.last() == contour.first() {
```